src/screens/staking-delegate-form.tsx

```diff
@@ -19,39 +20,42 @@ export const StakingDelegateFormScreen = () => {
   const wallet = Wallet.getById(account);
   const balances = useWalletsBalance([wallet!]);
   const currentBalance = useMemo(() => balances[account], [balances, account]);
-  const [fee, setFee] = useState<Balance | null>(null);
+  const [fee, _setFee] = useState<Balance | null>(null);
 
   const setDefaultFee = useCallback(
-    () => setFee(new Balance(Cosmos.fee.amount)),
+    () => _setFee(new Balance(Cosmos.fee.amount)),
     [],
   );
 
-  useLayoutEffectAsync(async () => {
-    const timer = setTimeout(() => {
-      setDefaultFee();
-    }, FEE_ESTIMATING_TIMEOUT_MS);
-
-    const instance = await getProviderInstanceForWallet(wallet!, true);
+  const setFee = useThrottle(async (amount?: string) => {
```

Comment:
We are hiding the error here. We need to understand why we have a lot of setState's in case of setFee